src/content/docs/codestream/how-use-codestream/discuss-code.mdx

```diff
@@ -177,4 +210,26 @@ Click the ellipses menu for any codemark and you'll see options to manage the co
 * **Edit**: Only the codemark's author can edit it.
 * **Delete**: Only the codemark's author can delete it, but we encourage you to archive instead of deleting unless you're positive the codemark won't have any future value.
 * **Inject as Inline Comment**: If you'd like a specific codemark to become part of the repo use this option to have it added as an inline comment. You can select the appropriate format, and then indicate if you want to include timestamps, replies, or to have the comment wrapped at 80 characters. You can also elect to have the codemark archived once it's been added as an inline comment.
-* **Reposition codemark**: In most cases, a codemark will automatically remain linked to the block of code it refers to as the file changes over time. For example, if you cut the block of code and paste it at a different location in the file, the codemark will move right along with it. There are some scenarios, however, that CodeStream isn't able to handle automatically. For example, if you pasted the block of code into a different file. In these cases, the **Reposition** codemark allows you to select the new location of the block of code so that the codemark is displayed properly.
\ No newline at end of file
+* **Reposition codemark**: In most cases, a codemark will automatically remain linked to the block of code it refers to as the file changes over time. For example, if you cut the block of code and paste it at a different location in the file, the codemark will move right along with it. There are some scenarios, however, that CodeStream isn't able to handle automatically. For example, if you pasted the block of code into a different file. In these cases, the **Reposition** codemark allows you to select the new location of the block of code so that the codemark is displayed properly.
+
+## Permalinks for sharing anywhere [#permalinks]
```

Comment:
The "Copy link" bullet in the list above now needs its anchor link updated to point to https://docswebsitedevelop-doc7773consolidatecodestreamui.gtsb.io/docs/codestream/how-use-codestream/discuss-code/#permalinks

Reply:
Done!